lightning/src/ln/channelmanager.rs

```diff
@@ -3028,7 +3029,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 										routing, incoming_shared_secret, payment_hash, amt_to_forward, outgoing_cltv_value },
 										prev_funding_outpoint } => {
 											macro_rules! fail_forward {
-												($msg: expr, $err_code: expr, $err_data: expr, $phantom_ss: expr) => {
+												($msg: expr, $err_code: expr, $err_data: expr, $phantom_ss: expr, $unknown_next_hop: expr) => {
```

Comment:
Boolean parameters aren't very readable. Could we refactor this into `failed_forward` and `failed_payment` macros?